tests/AutoReview/ProjectFixerConfigurationTest.php

```diff
@@ -26,22 +27,40 @@
  */
 final class ProjectFixerConfigurationTest extends TestCase
 {
-    public function testCreate()
+    /**
+     * @var \PhpCsFixer\Config
+     */
+    private $config;
+
+    protected function setUp()
     {
-        /** @var \PhpCsFixer\Config $config */
-        $config = require __DIR__.'/../../.php_cs.dist';
+        $file = __DIR__.'/../../.php_cs.dist';
+        $this->assertFileExists($file);
```

Comment:
assertions in set up? it doesn't look good

Reply:
not sure which way to refactor, any preference?

Reply:
replace whole body with
```php
$this->config = require __DIR__.'/../../.php_cs.dist';
```

if it's not the file, it will crash the `require` call, no need for assertion
if it won't return Config, usage of `$this->config` will crash, no need for assertion